### PR TITLE
Few new lines up to be localized from in battle.ts locales

### PR DESCRIPTION
**Pull request by Adri1**

### CODE REVIEW

**Review by GhostFlys — APPROVED** [review]

**Review by GhostFlys — APPROVED** [review]

File: src/locales/de/battle.ts [hunk 1]
```diff
@@ -12,7 +13,9 @@ export const battle: SimpleTranslationEntries = {
   "trainerGo": "{{trainerName}} sendet {{pokemonName}} raus!",
   "switchQuestion": "Möchtest du\n{{pokemonName}} auswechseln?",
   "trainerDefeated": `{{trainerName}}\nwurde besiegt!`,
+  "moneyWon": "You got\n₽{{moneyAmount}} for winning!",
   "pokemonCaught": "{{pokemonName}} wurde gefangen!",
+  "partyFull": "Your party is full.\nRelease a Pokémon to make room for {{pokemonName}}?",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:18]: "partyFull": "Dein Team ist voll.\nMöchtest du ein Pokémon durch {{pokemonName}} ersetzen?",

File: src/locales/de/battle.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const battle: SimpleTranslationEntries = {
   "bossAppeared": "{{bossName}} erscheint.",
   "trainerAppeared": "{{trainerName}}\nmöchte kämpfen!",
   "trainerAppearedDouble": "{{trainerName}}\nmöchten kämpfen!",
+  "trainerSendOut": "{{trainerName}} sent out\n{{pokemonName}}!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:7]: `  "trainerSendOut": "{{trainerName}} setzt\n{{pokemonName}} ein !",`

File: src/locales/de/battle.ts [hunk 3]
```diff
@@ -12,7 +13,9 @@ export const battle: SimpleTranslationEntries = {
   "trainerGo": "{{trainerName}} sendet {{pokemonName}} raus!",
   "switchQuestion": "Möchtest du\n{{pokemonName}} auswechseln?",
   "trainerDefeated": `{{trainerName}}\nwurde besiegt!`,
+  "moneyWon": "You got\n₽{{moneyAmount}} for winning!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:16]: `  "moneyWon": "Du gewinnst\n{{moneyAmount}} ₽!",`

File: src/locales/de/battle.ts (hunk 2 above)
Comment (CodeTappert) [reply src/locales/de/battle.ts:7]: Taken directly from the newest games

File: src/locales/de/battle.ts (hunk 3 above)
Comment (CodeTappert) [reply src/locales/de/battle.ts:16]: Taken directly from the newest games (thats why the money sign is behind)

File: src/locales/de/battle.ts (hunk 1 above)
Comment (CodeTappert) [reply src/locales/de/battle.ts:18]: This one doesnt exist in the game. So i translated it so it makes sense

**Comment by Adri1** [comment]
German translations added

File: src/locales/it/battle.ts [hunk 4]
```diff
@@ -12,7 +13,9 @@ export const battle: SimpleTranslationEntries = {
   "trainerGo": "{{trainerName}} manda in campo {{pokemonName}}!",
   "switchQuestion": "Vuoi cambiare\n{{pokemonName}}?",
   "trainerDefeated": `Hai sconfitto\n{{trainerName}}!`,
+  "moneyWon": "You got\n₽{{moneyAmount}} for winning!",
```
Comment (SavGRY) [inline src/locales/it/battle.ts:16]: The translation in Italian would be: `Hai vinto {{moneyAmount}}₽`

Comment (SavGRY) [reply src/locales/it/battle.ts:16]: Translation got by BDSP

**Review by SavGRY — COMMENTED** [review]
I've got some of the italian translation for the new added dialogues

**Comment by Adri1** [comment]
Suggested Italian translations added

**Comment by Adri1** [comment]
A acheck from an admin would be very welcome! 👌

**Comment by Adri1** [comment]
@bennybroseph It is possible to have review on this one or there is an issue I do not know?
Asking because for no apparent reason, I tend to see very recent localiztion PR being merged very quickly (or at least quicker), while some others like this one are still the queue for some days

**Comment by bennybroseph** [comment]
it's just hard to get to everything. with localization, things that don't change any game code can be merged pretty much immediately. things that add new localization take a review and sometimes when I look at it I don't have time to test

**Review by bennybroseph — APPROVED** [review]

**Pull request merged**